src/SFA.DAS.ApprenticePortal.Web/Pages/Privacy.cshtml.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace SFA.DAS.ApprenticePortal.Web.Pages
+{
+    public class PrivacyModel : PageModel
+    {
+        /// <summary>
+        /// Page yet to be implemented
+        /// </summary>
+        public void OnGet()
+        {
+        }
```

Comment:
Empty methods cause SonarCloud to complain.  As there is no logic in these pages we can actually just delete the `.cshtml` files completely.

Reply:
Removed code behind files